omnibus/config/software/libossp-uuid.rb

```diff
@@ -25,6 +25,10 @@
 build do
   env = with_standard_compiler_flags(with_embedded_path)
 
+  if version == "1.6.2" && ppc64le?
```

Comment:
I can't seem to find a difference between the libossp-uuid definition here and the one in omnibus software.  Perhaps we should make these changes in the omnibus-software definition and just remove this copy.